### PR TITLE
Add support and testing for new Django and Python versions

### PR DESCRIPTION
**Pull request by jdufresne**
Django 3.1 was released August 4, 2020.

Python 3.9 was released October 5th, 2020.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -68,7 +68,8 @@ Django versions Python versions
 2.0             3.4, 3.5, 3.6, 3.7
 2.1             3.5, 3.6, 3.7
 2.2             3.5, 3.6, 3.7, 3.8
```
Comment (twidi) [inline README.rst:70]: +3.9

File: tox.ini [hunk 2]
```diff
@@ -5,8 +5,9 @@ envlist =
     py{34,35,36,37}-django20
     py{35,36,37}-django21
     py{35,36,37,38}-django22
```
Comment (twidi) [inline tox.ini:7]: +`,39`

File: README.rst [hunk 3]
```diff
@@ -86,7 +87,8 @@ Django versions Python versions         Library versions
 2.0             3.4, 3.5, 3.6, 3.7      3.0
 2.1             3.5, 3.6, 3.7           3.0
 2.2             3.5, 3.6, 3.7, 3.8      3.0
```
Comment (twidi) [inline README.rst:89]: +`, 3.9`

**Review by twidi — CHANGES_REQUESTED** [review]
You missed to add python 3.9 for django 2.2 at some places

**Comment by jdufresne** [comment]
Thanks for catching that. Fixed in the latest revision.

**Review by twidi — APPROVED** [review]

**Pull request merged**